include/nbl/builtin/glsl/ies/functions.glsl

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018-2024 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_GLSL_IES_FUNCTIONS_INCLUDED_
+#define _NBL_GLSL_IES_FUNCTIONS_INCLUDED_
+
+#include <nbl/builtin/glsl/math/constants.glsl>
+
+vec2 nbl_glsl_IES_convert_dir_to_uv(vec3 dir) {
```

Comment:
write in the comments what I actually asked for as a TODO

Reply:
basically this as a comment https://github.com/Devsh-Graphics-Programming/Nabla/pull/630#discussion_r1431395101